packages/payment-detection/test/near/near-native.test.ts

```diff
@@ -113,6 +113,46 @@ describe('Near payments detection', () => {
     expect(balance.balance).toBe('1000000000000000000000000');
   });
 
+  it('NearNativeTokenPaymentDetector can detect a payment on Near with an additional declarative payment', async () => {
```

Comment:
This test is not related at all! But since it was absent, I thought for a moment that it did not work and was linked to the issue.